modules/ROOT/pages/design-publish.adoc

```diff
@@ -1,59 +1,63 @@
+* If the specification is being republished and you select the `In Development` state, you can can use the same number as a previously-published in-development version of the API, but not the same number as a previously-published stable version.
 
-== Filling in the *API version* Field
+[[api-version]]
+== API Version
 
-The appearance of the *API version* field depends upon several factors:
+Anypoint Exchange requires an API version number for each API specification published. 
 
-* Anypoint Exchange requires that each API specification published to it have a version number.
-* API Manager requires that API specifications provide a version number.
-* In specifications written in OAS, the `version` field is required.
-* In specifications written in RAML, the `version` node is optional.
+The content and state of the *API version* field on the publish dialog depends on the following:
 
-The following flowchart shows how API Designer decides whether to disable or enable the *API version* field and whether to populate it with a number:
+* If the specification includes the version, the version is filled in from the specification and you cannot update it on the publish dialog.
+* If the specification does not include the version, you must enter the version in the *API version* field. 
 
-.The appearance of the *API version* field depends on the outcome of the checks that API Designer makes when you want to publish an API specification.
-image:apid-flowchart-api-version.png[Flowchart for the checks that API Designer makes]
+== See Also
 
-<1> API Designer checks whether the version number for the API specification is provided within the specification. If the version number is provided, then the *API version* field is populated with the number and disabled, so that you cannot change the number. If the specification does not provide the version number, then API Designer moves to the next check.
-<2> API Designer checks whether the checkbox *Add to API Manager* is selected. If the checkbox is selected, then API Designer prevents you from proceeding because API Manager requires that the specification provide the version number. If the checkbox is not selected, then API Designer moves to the last check.
-<3> API Designer checks whether the specification is being published to Anypoint Exchange for the first time. If the specification is being published for the first time, then API Designer enables the *API version* field and suggests a version number because a number is not provided in the specification. If the specification is not being published for the first time, then API Designer enables the *API version* field and populates the field with the number used when the specification was last published. You can change the number.
+* xref:exchange::workflow-sharing-assets.adoc[Workflow: Publish Assets]
```

Comment:
I used the name that is actually in the Exchange docs, but will change it here, since I can use whatever text I want for the cross-reference display name. I will give Jenny a heads up about it.